src/routes/TriggerMessage/component.js

```diff
@@ -20,7 +20,7 @@ export default class TriggerMessage extends Component {
 	componentDidUpdate() {
 		let height = 0;
 
-		this.ref.current.base.children.forEach((el) => {
+		[...this.ref.current.base.children].forEach((el) => {
```

Comment:
Do you need the spread here? 👀

Reply:
yes, forEach is undefined without it for some reason

Reply:
so... children is not an iterable array type indeed...

Reply:
@cauefcr is doing almost the same as Array.from(childrenEls), that's enough to me...

Reply:
Yep, the thing is that ... and .foreach should behave similar for arraylike elements (like children lists) 👀 that's why i was wondering if it was needed haha